devito/arch/compiler.py

```diff
@@ -761,10 +761,6 @@ def __init__(self, *args, **kwargs):
         platform = kwargs.pop('platform', configuration['platform'])
         language = kwargs.pop('language', configuration['language'])
 
-        if language == 'openmp':
```

Comment:
There is no guaranty an generic user will have icx >= 2023.2 the version number needs to be checked

Reply:
Right, thanks! I was thinking of our docker only! WIll fix